config/environments/development.rb

```diff
@@ -14,9 +14,11 @@
   config.action_controller.perform_caching = false
   config.serve_static_files                = false
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  # Do care if the mailer can't send.
+  config.action_mailer.delivery_method = :letter_opener
```

Comment:
Go ahead and revert changes to this file as well -- we can make this work without raising delivery errors or using `letter_opener`.